liveblog-embed/resources.service.js

```diff
@@ -91,7 +91,14 @@
     Posts.$inject = ['$resource', 'config', 'users', 'srcSet', 'fixProtocol'];
     function Posts($resource, config, users, srcSet, fixProtocol) {
         function _completeUser(obj) {
-            if (obj.commenter) {
+            if (obj.syndication_in && obj.mainItem) {
```

Comment:
After testing it is not raising an issue, I guess syndication_in is being removed